src/utils/queryBuilder.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+// Builds up filter string by filter type
+const eventFilters = new Map([
+  ['startDate', val => `start_date gt ${moment(val).format('YYYY-MM-DD')}`],
```

Comment:
Is it clear to the user that the date select is a start date? I imagine the expectation is that it equals the date selected.